doc/configuration.rst

```diff
@@ -411,6 +411,35 @@ hive resources and 1 mysql resource:
 Note that it was not necessary to specify the 1 for mysql here, but it
 is good practice to do so when you have a fixed set of resources.
 
+.. _retcode-config:
+
+[retcode]
+----------
+
+Configure return codes for the luigi binary. In the case of multiple return
```

Comment:
What use is there for making them configurable? An unusual enabler of accidental complexity.

Perhaps declare in documentation a "reserved" range of return codes that Luigi might define later, the rest being available for application-specific usage.